scripts/setup-ssh-keys.sh

```
@@ -0,0 +1,767 @@
+        
+        # Use appropriate port option
+        local scan_opts=""
+        if [ "$port" != "22" ]; then
+            scan_opts="-p $port"
+        fi
+        
+        echo -n "Scanning keys for $host_name ($hostname:$port)... "
+        print_verbose "Running: timeout 10 ssh-keyscan -H ${scan_opts:-} $hostname"
+        
+        # Scan and add to known_hosts (with 10 second timeout)
+        if timeout 10 ssh-keyscan -H $scan_opts "$hostname" >> ~/.ssh/known_hosts 2>/dev/null; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)